### PR TITLE
Feature/register

### PR DESCRIPTION
**Pull request by JessMotta**
- creation of function in API to associate user id with category id
- script creation to read the information in the form fields and create a new user with associated category

### CODE REVIEW

File: css/cards.css [hunk 1]
```diff
@@ -1,25 +1,28 @@
 .card {
   background-color: var(--color-white);
   width: 234px;
+  /* max-width: fit-content; */
```
Comment (luanaggoncalves) [inline css/cards.css:4]:
```suggestion
```

File: pages/register.html [hunk 2]
```diff
@@ -80,7 +83,7 @@ <h1><img src="../assets/images/logo.png" alt=""></h1>
                                 <option value="node-js">Node.js</option>
                             </select>
                         </div>
-                    </div>
+                    </div> -->
```
Comment (luanaggoncalves) [inline pages/register.html:86]: Podemos remover se não for mais utilizado

```suggestion
```

File: css/cards.css (hunk 1 above)
Comment (JessMotta) [reply css/cards.css:4]: removed

File: pages/register.html (hunk 2 above)
Comment (JessMotta) [reply pages/register.html:86]: removed

**Pull request merged**